cas/cas.h

```diff
@@ -135,6 +135,21 @@ struct m0_cas_kv_vec {
 	struct m0_cas_kv *cv_rec;
 } M0_XCA_SEQUENCE M0_XCA_DOMAIN(rpc);
 
+/**
+ * Version of a CAS record.
+ * A version comprises a physical timestamp and a tombstone flag.
+ * The timestamp is set on the client side whenever the corresponding
+ * record was supposed to be modified by PUT or DEL request.
+ * The tombstone flag is set when the corresponding record has been
+ * "logicaly" removed from the storage (although, it still exists
+ * there as a "dead" record).
+ * See ::COF_VERSIONED for details.
+ */
+struct m0_cas_kv_ver {
+	struct m0_dtm0_ts ckv_ts;
+	bool              ckv_tombstone;
```

Comment:
a very strange question...
do you think we should avoid using `bool` types in network messages for any reason?

Reply:
That's a good question. Generally speaking, we should not use non-fixed-size types for on-disk/on-wire data types. However, in our case, the platform is fixed, so xcode should always generate the same u8 parsers for this field.

Nevertheless, I believe the idea of exposing versions/tombstones using a separate type did not serve its purpose, so I just removed this structure and now we use m0_crv everywhere (see https://github.com/Seagate/cortx-motr/pull/889/commits/262f68da0c955caa0674a60222fd947e52571724)